benchmarks/pytest-based/bench_algos.py

```diff
@@ -366,7 +380,9 @@ def bench_in_degree_centrality(benchmark, graph_obj, backend_wrapper):
 def bench_katz_centrality(benchmark, graph_obj, backend_wrapper, normalized):
     G = get_graph_obj_for_benchmark(graph_obj, backend_wrapper)
     result = benchmark.pedantic(
-        target=backend_wrapper(nx.katz_centrality),
+        target=possible_to_fail(
+            nx.PowerIterationFailedConvergence, backend_wrapper(nx.katz_centrality)
+        ),
```

Comment:
Although I think the code is pretty self-explanatory, it may still be helpful to add a comment why "possible-to-fail" is acceptable here. That is, both networkx and nx-cugraph fail to converge for some graphs, which is a perfectly acceptable benchmark imho.